### ISSUE TITLE
Browsersync v{{app.options.version}}

### ISSUE THREAD

**Issue by lordisp** [maizzle/framework#564]
- Maizzle Version: v3.7.2
- Node.js Version: v14.15.4

steps to reproduce:
* running `maizzle serve`
* brows http://localhost:3001
* getting below

```
✔ Re-built 2 templates in 0.575s               
[Browsersync] Access URLs:                     
 ----------------------------------            
       Local: http://localhost:3000            
    External: http://10.0.0.6:3000             
 ----------------------------------            
          UI: http://localhost:3001            
 UI External: http://localhost:3001            
 ----------------------------------            
[Browsersync] Serving files from: build_local  
```

![image info](https://db3pap001files.storage.live.com/y4mwzUhCMHKvY0eOJKH27kFgNrRhPremYpyBZ-Oq4bSfgbeoSMkrPVNObJXLBf6soC53hSMsAwe1yvxlNUgHZ7FjIZ-QD0ZJPawuOK7auGuVBeBkJQnNlwdiA26wF0OGzYP5D9ddC6zrc3WYAR1LCPbM4k0Rl2dRVlXuflt4XEv_ilzl_ZbBgH3ntdTzJDzkEYu?width=479&height=277&cropmode=none)

**Comment by cossssmin** [comment]
Looks like it's an issue with Browsersync, need to wait for them to release a fix 👍

https://github.com/BrowserSync/browser-sync/issues/1917

**Comment by anonrig** [comment]
This is still an issue with the @maizzle/cli

**Comment by cossssmin** [comment]
> This is still an issue with the @maizzle/cli

This is still an issue with BrowserSync, it'll be fixed in Maizzle when they release a fix.

**Comment by cossssmin** [comment]
It uses`v2.27.7`, which is the latest. The issue for this bug is still open, it's linked right above.

**Comment by cossssmin** [comment]
Fixed in 90776e17b7c6a05b200300ee39270bf79c23c79b

![image](https://user-images.githubusercontent.com/1656595/158226291-23c4aaff-3ea1-4df8-8749-5ec04aff3776.png)
